Screens/ImageCapture.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import Camera from 'react-native-camera'
+
+class ImageCapture extends Component {
+  state = {
+    images: []
+  }
+
+  static navigationOptions = {
+    title: 'Camera'
+  }
+
+  takePicture() {
+    const options = {}
+    this.camera.capture({ metadata: options })
```

Comment:
>capture [](start = 16, length = 7)

Perhaps you could change the options to use { target: 'disk' } or { target: 'cameraRoll' }. It's not super efficient to pass base64 encoded photos across bridge and store them in memory in state in JS. This will likely end up on a less frequently collected heap in JS, so you could be holding some binary photo data in memory for quite some time.

Reply:
I believe react-native even supports blobs now, so you never have to transfer the binary photo data to JS (it always stays native) and you can point the XHR request to the blob ID or file location to upload the photo.

---
In reply to: [157104235](https://github.com/rozele/React-Native-Ski-Team/pull/2#discussion_r157104235) [](ancestors = 157104235)

Reply:
Note - I could be wrong here - I'm not sure what the default behavior of react-native-camera is, I'm assuming that the default is to capture the photo in memory.

---
In reply to: [157104373](https://github.com/rozele/React-Native-Ski-Team/pull/2#discussion_r157104373) [](ancestors = 157104373,157104235)